test/examples/deviceProvisioning/expectedProvisioningPluginRequest.json

```diff
@@ -8,35 +8,35 @@
               {
                   "name": "time",
                   "type": "String",
-                  "value": " "
+                  "value": null
```

Comment:
Currently unit tests in master are failing:

![imagen](https://user-images.githubusercontent.com/1534240/103891453-277ced00-50ea-11eb-9140-bfeef828b94c.png)

Are the changes `" "` to `null` included in this PR the fixes for that?

Just to confirm...

Reply:
Yes

Reply:
NTC